frontend/best_practices.md

```diff
@@ -91,7 +91,7 @@ export const RobotComponent = ({ mission, setColor } : {mission: MissionDefiniti
 
 ### React state
 
-React provides the useState hook which allows the user to define variables which persist between renders. Although it is tempting to use these for all variables, it is worth noting that unless the variable has some information which needs to be kept for another render in order to update another variable, it can be kept as a normal Typescript variable.
+React provides the useState hook which allows the user to define variables that persist between renders. Although it is tempting to use these for all variables, it is worth noting that unless the variable has some information which needs to be kept for another render in order to update another variable, it can be kept as a normal Typescript variable.
```

Comment:
```suggestion
React provides the useState hook which allows the user to define variables that persist between renders. Although it is tempting to use these for all variables, it is worth noting that unless the variable has some information which needs to be kept for another render in order to update another variable, it can be kept as a normal TypeScript variable.
```